index.bs

```diff
@@ -7580,6 +7580,68 @@ To <dfn abstract-op>Create a new supplemental public key record</dfn>, perform t
         [=set/append=] this [=supplemental public key record=] to |credentialRecord|.[$credential record/supplementalPubKeys$].
 
 
+### User verification caching extension (<dfn>userVerificationCaching</dfn>) ### {#sctn-user-verification-caching-extension}
+
+In some cases it is good enough for the [=RP=] to know whether 
+    the user was verified by the authenticator "recently".
+    This extension allows the [=RP=] to specify such [=user verification=]
+    caching time, i.e. the time for which the
+    [=user verification=] status can be "cached" by the [=authenticator=].
+
+For example: Do not ask the user for a fresh [=user verification=] for sign-in 
+    if the user was verified by this authenticator within the past 300 seconds.
+
+#### Extension Definition #### {#sctn-user-verification-cacing-extension-definition}
+
+: Extension identifier
+:: `userVerificationCaching`
+
+: Operation applicability
+:: [=authentication extension|authentication=]
+
+: Client extension input
+:: The maximum [=user verification=] caching time denotes the maximum acceptable number of seconds elapsed since the last time the user was successfully verified.
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientInputs {
+      unsigned short maxUVC;
+    };
+    </xmp>
+
+: Client extension processing
+:: None, except creating the [=authenticator extension input=] from the client extension input.
+
+: Client extension output
+:: Returns a JSON object denoting the maximum acceptable number of seconds elapsed since the last time the user was successfully verified as returned by the [=authenticator=].
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+      unsigned short maxUVC;
```

Comment:
Ditto:
```suggestion
:: Returns a JSON object denoting the maximum acceptable number of milliseconds elapsed since the last time the user was successfully verified as returned by the [=authenticator=].
    <xmp class="idl">
    partial dictionary AuthenticationExtensionsClientOutputs {
      unsigned short userVerificationCaching;
```

Reply:
Also, this output is not a JSON object, just an integer. I think we can just reference the authenticator output here.

```suggestion
:: The authenticator extension output parsed as an integer value.
    <xmp class="idl">
    partial dictionary AuthenticationExtensionsClientOutputs {
      unsigned long userVerificationCaching;
```